GeneratingDataset.py

```diff
@@ -1589,7 +1609,7 @@ def get_seq(self, sentence):
     :rtype: list[int]
     """
     segments = self._segment_sentence(sentence)
-    seq = self.get_seq_indices(segments)
+    seq = super(BytePairEncoding, self).get_seq_indices(segments)
```

Comment:
Why is the `super` needed here?

Reply:
Because get_seq_indices() only exists in the parent Vocabulary class. If just writing "self" would also work, I can remove it.

Reply:
Yes of course, just `self` works. Anything related to this code also has not changed, `get_seq_indices` was always part of the parent class.